config/busola/logpipeline_busola_extension_cm.yaml

```diff
@@ -445,7 +633,8 @@ data:
     - source: >-
```

Comment:
how about we simplify the `source` to something like this:
```
source: >-
  $each(spec.output, function($v, $k) { $exists($v) ? $k })
```

and in this case we can remove the `custom` placeholder

Reply:
done